src/commands/group/stop.js

```diff
@@ -27,7 +27,9 @@ class GroupStopCommand extends GroupBaseCommand {
         {
           title: `Stop ${groupName} nodes`,
           task: () => (
-            new Listr(configGroup.map((config) => ({
+            // So we stop the miner first, as there's a chance that MNs will get banned
+            // if the miner is still running when stopping them
+            new Listr(configGroup.reverse().map((config) => ({
```

Comment:
There is no guarantee that it will get always `local_1`, `local_2`, `local_seed` order. I would recommend sorting or manually setting the seed node as the first one.